### DISCUSSION TITLE
I do not see the "Trusted" checkbox despite being the only admin

### DISCUSSION THREAD

**Discussion by wsw70** [woodpecker-ci/woodpecker#3361] [Ask a Question] [answered]
I installed Woodpecker for my use (as an amateur dev) and I have a single account on the server. I am therefore admin.

I still cannot see the "Trusted" setting for my project despite what the docs mentions:

> Only server admins can set this option. If you are not a server admin this option won't be shown in your project settings

I just see

![image](https://github.com/woodpecker-ci/woodpecker/assets/1345886/441d0917-2cf1-4457-acb3-505d17d663ba)

Is there something else I should set up?

**Comment by anbraten** [comment, marked as answer]
Have you set yourself as admin via `WOODPECKER_ADMIN`?

**Reply by wsw70** [reply]
> Have you set yourself as admin via `WOODPECKER_ADMIN`?

No I did not. I was under the (false) impression that the first account would also be admin. Thank you very much for the information!

**Reply by anbraten** [reply]
If you have any suggestions how we can improve the docs in this aspect, please feel free to open a PR.

**Reply by moedesux** [reply]
I have the same problem too. I'm selfhosting Gitea and Woodpecker. I set WOODPECKER_ADMIN=MY_GITEA_REPO_USERNAME in the woodpecker-server service in my docker compose as below (I trimmed the rest for brevity).


  woodpecker-server:
    container_name: woodpecker-server
    image: woodpeckerci/woodpecker-server:v3
    environment:
      - WOODPECKER_ADMIN=MY_GITEA_REPO_USERNAME
      
 I'm running Woodpecker 3.8.0

**Reply by qwerty287** [reply]
Do you have access to the admin settings?

**Reply by moedesux** [reply]
Yes I have. Here how it looks like

<img width="1032" height="325" alt="image" src="https://github.com/user-attachments/assets/ac89a871-b51c-4c29-979a-45db569ea8c8" />

and here is my repo settings

<img width="994" height="890" alt="image" src="https://github.com/user-attachments/assets/0297e2b1-7cda-4a08-97ed-ef107fecba02" />

**Reply by qwerty287** [reply]
Sorry, but please just check the interface. There is the "trusted" section. There is no single trusted checkbox anymore but it's split into three.

**Reply by moedesux** [reply]
Aha, thanks for pointing out. I was following the official guide [here](https://woodpecker-ci.org/docs/usage/project-settings) but seems it is not up-to-date. Much appreciated
